qemu/tests/create_external_snapshot_via_driver.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import re
+import shutil
+
+from autotest.client.shared import error
+
+from avocado.core import exceptions
+from avocado.utils import process
+
+from qemu.tests import qemu_disk_img_info
```

Comment:
Why use qemu_disk_img_info, i think we can import qemu_disk_img directly. I have submitted a patch move function check_backingfile() to parent class.